packer/README.md

```diff
@@ -2,14 +2,36 @@
 
 This directory contains configuration for running packer to build AMIs.
 
+## Dependencies
+
+Running these packer scripts requires the following software:
+
+- python3
+- [packer](https://developer.hashicorp.com/packer/downloads)
+- [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+
 ## The `packer` wrapper
 
 This directory contains a python script named `packer` which wraps the system `packer`. The script creates an ansible virtual environment and moves the correct ansible configuration in place.
 
 ### Running `packer`
 
-To run the wrapper pass the environment and playbook:
+Before running the packer script, you will need to initialize packer:
+
+```bash
+packer init ./docs-rs-builder
```

Comment:
This needs to be run from within the `packer` directory, correct? If so, do you want to add a `cd packer` or so to the snippet?